test/helpers/compiler.js

```diff
@@ -8,6 +8,8 @@ import path from 'path';
 import webpack from 'webpack';
 import MemoryFS from 'memory-fs';
 
+const majorVersion = require('webpack/package.json').version.split('.')[0];
```

Comment:
`majorVersion` => `version`

Do we need `mode` for testing ?

Reply:
Unless you want to not ever be able to run the test suite on 3.x & 4.x

Reply:
`majorVersion` => `version` : It is exactly what it's doing. It's not checking the version, it's grabbing the number that represents the major version.